v1/lib/cell.php

```diff
@@ -1,25 +1,28 @@
 <?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
 
 /**
- * Created by PhpStorm.
- * User: Max
- * Date: 15.01.2016
- * Time: 12:01
- *
- * Class of a single cell of the the gamefield!
+ * This class is a cell object, every instance of this class is one cell
```

Comment:
This class represents a cell, which contains information the information about whether this cell is alive or dead.